modules/ROOT/pages/_partials/sync/getting-started-openshift.adoc

```diff
@@ -0,0 +1,73 @@
+= Prerequisites
+* [x] You have Docker installed.
+* [x] You have access to an OpenShift cluster with the Service Catalog.
+* [x] You have completed the server getting started guide.
+
+= Overview
+
+To connect your Data Sync server and client to other mobile services, you need to run your application in OpenShift.  Data Sync provides a service catalog item to help with this.
```

Comment:
>  you need to run your application in OpenShift.

Really strange to read this. We want to hide the fact that you could run this locally with no problem whatsoever?

Reply:
no, but this section is about getting started on openshift. The local dev part is covered in both data sync server and client sections